src/features/Footer/footer.module.styl

```diff
@@ -17,10 +17,15 @@
   p
     color $ftr-secondary
   a
-    color $ftr-secondary
+    color $ftr-primary
+    font-size 14px
+    font-weight 300
+    &:hover
+      color $ftr-hover
 
 .solutions
-  max-width 209px
+  max-width 260px
+  padding-top 22px
   h5
```

Comment:
This header was changed from a h5 to an h4. Are the styles still relavent or can they be removed?